.circleci/scripts/git-diff-develop.ts

```diff
@@ -58,14 +58,10 @@ async function fetchUntilMergeBaseFound() {
  * It first ensures that the necessary commits are fetched until the merge base is found.
  *
  * @returns The output of the git diff command, listing the changed files.
- * @throws If unable to get the diff after fetching the merge base or if an unexpected error occurs.
  */
 async function gitDiff(): Promise<string> {
   await fetchUntilMergeBaseFound();
   const { stdout: diffResult } = await exec(`git diff --name-only origin/HEAD...${process.env.CIRCLE_BRANCH}`);
-  if (!diffResult) {
```

Comment:
if the git diff is empty, means that there is no difference in any of the files. In this case, we should not throw an error, since this is the case for develop branch (git diff develop develop, will be empty)